app-modules/license-management/database/migrations/2024_11_21_194048_data_seed_permissions_for_product_license_module.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Aiding App™ is licensed under the Elastic License 2.0. For more details,
+    see <https://github.com/canyongbs/aidingapp/blob/main/LICENSE.>
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\Migrations\Migration;
+use Database\Migrations\Concerns\CanModifyPermissions;
+
+return new class () extends Migration {
+    use CanModifyPermissions;
+
+    private array $permissions = [
+        'product.view-any' => 'Product',
+        'product.create' => 'Product',
+        'product.*.view' => 'Product',
+        'product.*.update' => 'Product',
+        'product.*.delete' => 'Product',
+        'product.*.restore' => 'Product',
+        'product.*.force-delete' => 'Product',
+        'product-license.view-any' => 'Product License',
+        'product-license.create' => 'Product License',
+        'product-license.*.view' => 'Product License',
+        'product-license.*.update' => 'Product License',
+        'product-license.*.delete' => 'Product License',
+        'product-license.*.restore' => 'Product License',
+        'product-license.*.force-delete' => 'Product License',
```

Comment:
Current permission convention uses `_` for the first part of the permissions.

So this should be `product_license.view-any` and so on.